tests/test_command.py

```diff
@@ -798,3 +798,15 @@ def test_help_text(self):
                             # not too long
                             assert len(help_text) < 80
         assert not commands, "Commands without help text: %s" % commands
+    
```

Comment:
**mike bayer** (zzzeek) wrote:

looks like trailing whitespace

to format everything perfectly:

    pip install pre_commit
    pre-commit install

this will run black on the commit